lib/src/view/menu.dart

```diff
@@ -1040,6 +1040,10 @@ debugging, but be warned that it will be very slow to render a large number of D
         ..on_click = ((_) => app.disable_keyboard_shortcuts_while(export_dna))
         ..tooltip = "Export DNA sequences of strands to a file."
         ..display = 'DNA sequences')(),
+      (MenuDropdownItem()
+        ..on_click = ((_) => props.dispatch(actions.ExportCanDoDNA(export_dna_format: ExportDNAFormat.cando)))
+        ..tooltip = "Export DNA sequences of strands as a CSV for use with CanDo."
```

Comment:
I'm not exactly sure how to use the exported file. The last time I used CanDo online (like 10 years ago), it required a cadnano file: http://cando-dna-origami.org/cadnano-submission/#

Is this for the offline version of CanDo? It might be good to include in the tooltip here a URL link to a page with instructions on what to do with this file. e.g., here? https://github.com/lcbb/Off-Lattice-CanDo... I actually can't find anywhere a description of the CSV file format being exported here

Reply:
Hello! CanDo now supports the cadnano2.json format and an optional .csv if you check 'Would you like the atomic model included with your results?'.

Reply:
Do my changes resolve this? @dave-doty